CliClient/tests/services_KeymapService.js

```diff
@@ -0,0 +1,279 @@
+require('app-module-path').addPath(__dirname);
+
+const { shim } = require('lib/shim');
+const KeymapService = require('lib/services/KeymapService.js');
+const keymapService = KeymapService.instance();
+
+describe('services_KeymapService', () => {
+	describe('validateAccelerator', () => {
+		it('should identify valid Accelerators', () => {
+			const testCases = shim.isMac()
```

Comment:
Tests should not be platform-specific, because if you make changes to one platform code, the tests might pass even though something is broken in another platform. It's best to pass the platform name to the KeymapService constructor for example. You can default the value to shim.platformName() if you want, but it should be possible to set it to something else.

This comment applies to all the uses of shim.isMac in the tests.

Reply:
Thanks for pointing that out. I faced this exact situation situation after splitting the keymap into macOS and other platforms. What I did was either use the CI to see if tests had been passed, or boot up macOS in a VM and perform tests manually.

Providing a method to change the platform will enable it to test for both platforms at once. I'll come up with a solution to this.

Reply:
Yes when we build the apps on CI (Travis and Appveyor) we could technically run platform-specific tests, however it's nice too to be able to run them all locally, and for that the easiest way (at least in this case) is to provide the platform to the service.

By the way, rather than changing the keymap based on a boolean (isMac) please make it change based on a string (platformName). So for now, your code would be very similar, just checking for platformName = 'darwin', but later we also have the option to add support for other platform-specific shortcuts.